js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+                'createOrder': true,
+                'cancelOrder': true,
+                'cancelAllOrders': true,
+            },
+            'timeframes': {
+                '1m': '1m',
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '4h': '4h',
+                '12h': '12h',
+                '1d': '1d',
+                '1w': '1w',
+            },
+        });
+    }
+
+    async fetchMarkets (params = {}) {
+        let response = await this.publicGetMarkets ();
+        const result = [];
+        const markets = this.safeValue (response, 'data');
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            let id = market['id'].toUpperCase ();
+            let symbol = market['name'];
+            let pair = market['name'].split ('/');
+            let base = pair[0];
+            let quote = pair[1];
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': base,
+                'quoteId': quote,
+                'active': true,
+                'info': market,
+                'precision': undefined,
+                'limits': undefined,
+            });
+        }
+        // parse markets
+        return result;
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        let response = await this.publicGetTickersPair (this.extend ({ 'pair': symbol }), params);
+        const data = response['data'];
+        return this.parseTicker (data, symbol);
+    }
+
+    parseTicker (data, market = undefined) {
+        let timestamp = data['at'];
+        const ticker = data['ticker'];
+        if (typeof timestamp === 'string') {
+            if (timestamp.length > 0) {
+                timestamp = this.parse8601 (timestamp);
+            }
+        }
+        return {
+            'symbol': market,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeFloat (ticker, 'high'),
+            'low': this.safeFloat (ticker, 'low'),
+            'bid': this.safeFloat (ticker, 'buy'),
+            'bidVolume': undefined,
+            'ask': this.safeFloat (ticker, 'sell'),
+            'vwap': undefined,
+            'open': undefined,
+            'close': this.safeFloat (ticker, 'last'),
+            'last': this.safeFloat (ticker, 'last'),
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeFloat (ticker, 'volume'),
+            'quoteVolume': undefined,
+            'info': ticker,
+        };
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        let response = await this.publicGetOrderBook (this.extend ({
+            'market': symbol,
```

Comment:
We need to pass not the unified symbol, but the exchange-specific `market['id']` here and everywhere throughout this file in order to keep it in line with other exchanges, see an example here: https://github.com/ccxt/ccxt/blob/master/js/hitbtc2.js#L741